### DISCUSSION TITLE
Inherit KeyFrame and TranslateTransform for customization

### DISCUSSION THREAD

**Discussion by SwiftExtender** [AvaloniaUI/Avalonia#15862] [Q&A]
Inherit KeyFrame and TranslateTransform for customization

I have project with Avalonia 0.11 and i need to upgrade to Avalonia 11.0.10. The main problem with KeyFrame and TranslateTransform, because they are customized by inheriting  and they are sealed in Avalonia 11.0.10. 

How to customize KeyExt and TranslateTransform in Avalonia 11? There are no hints in "Breaking changes" issue, please help

**Comment by maxkatz6** [comment]
What is KeyExt? And what are you trying to solve by inheriting these types?

**Comment by SwiftExtender** [comment]
@maxkatz6 Excuse me, it was a typo. I meant "KeyFrame".
 I am trying to solve the customization task of view layer for custom control, so i can use customed styledProperty which creates customed properties for KeyFrame, which can be displayed by various means

Sorry, English is not my native language.
